src/CattyTests/PlayerEngine/Functions/Text/RegularExpressionFunctionTest.swift

```diff
@@ -55,7 +61,6 @@ class RegularFunctionTest: XCTestCase {
 
         XCTAssertEqual("", function.value(firstParameter: " am? a [^ .]+ ([a-z]+)" as AnyObject, secondParameter: "My number is 9999999999" as AnyObject))
 
-        XCTAssertEqual("The value “([0-9]{10}” is invalid.", function.value(firstParameter: "([0-9]{10}" as AnyObject, secondParameter: "My number is 9999999999" as AnyObject))
```

Comment:
Sorry, I did not see that there was already an assert for this case. Thanks for adding the new test method, now it is clearly readable.

Reply:
If everything is all right then I will squash and push.

Reply:
Please wait with squashing, there is one more thing we need to address (from a "feature point of view"). Will add an comment soon.